lib/results.js

```diff
@@ -22,6 +22,7 @@ function Results () {
     this.pass = 0;
     this._stream = through();
     this.tests = [];
+    this._only = 0;
```

Comment:
this can probably be initialized to `null`, or `function () {}`, rather than zero?